components/TheFooter.vue

```diff
@@ -1,90 +1,142 @@
 <template>
-    <footer class="bg-brand-grey">
-        <div class="container flex justify-around mx-auto p-4">
-            <a
-                class="link white-60 bg-transparent hover-white inline-flex items-center ma2 tc br2 pa2"
-                href="https://facebook.com/devedmontonsociety"
-                title="Dev Edmonton Society Facebook"
+    <!-- This example requires Tailwind CSS v2.0+ -->
+    <footer class="bg-grey-000">
```

Comment:
I really like the addition of some of the navigation to the footer. Nice work here @dboydgit